packages/core/types/src/fulfillment/mutations/shipping-option.ts

```diff
@@ -87,7 +87,7 @@ export interface UpdateShippingOptionDTO {
    * The shipping option type associated with the shipping option.
    */
   type?:
-    | Omit<CreateShippingOptionTypeDTO, "shipping_option_id">
+    | CreateShippingOptionTypeDTO | string
```

Comment:
suggestion: I think there is a discrepency between the two interface, let me know wyt:

in the create interface we accept for a type the DTO | string, but for the update we accept DTO | string | { id }. wdyt of aligning both by either changing the craete to DTO | { id } or changing the update by removing the { id }. I might be missing some context but I just wanted to raise the point 🙈